template/scripts/prepForBuild.js

```diff
@@ -9,7 +9,7 @@ let copyFiles = async function() {
     console.log(`${bundle} was copied to ./build dir`);
   });
 
-  fs.copyFile(`./demo/css/style.css`, `./build/css/style.css`, (err) => {
+  fs.copyFile(`./demo/style.css`, `./build/style.css`, (err) => {
```

Comment:
I think this needs to account for:

```suggestion
  fs.copyFile(`./demo/sass/style.scss`, `./build/style.scss`, (err) => {
```

Reply:
This is being removed because those sub-directories don't exist anymore? Refer to the [error](https://github.com/AlaskaAirlines/WC-Generator/issues/160) that prompted the change.

Reply:
Some old repos still have `./demo/sass/style.scss' - shouldn't we be preserving that file?